### DISCUSSION TITLE
Determining Scroll to the End

### DISCUSSION THREAD

**Discussion by chandramuralis** [stephanrauh/ngx-extended-pdf-viewer#2601] [Ideas]
Does this library provides any way to determine if the user have scrolled to the end of the page and fire some kind of event once they reached to the bottom? As a legal requirement we have to make sure that the user has scroll all the way to the end.

I couldn't find any such event out of the box based on the documentation, but wondering if anyone have figured out how to achieve this.

**Comment by stephanrauh** [comment]
What about using `(pageChanged)`? If it's a legal document, it probably consists of many pages, so checking if the user has reached the last page sounds like a good compromise.

**Reply by chandramuralis** [reply]
Yep, that's what I'm actually trying now. Thanks for the suggestion. Will post my finding here.

**Reply by chandramuralis** [reply]
> Yep, that's what I'm actually trying now. Thanks for the suggestion. Will post my finding here.

This worked perfectly.

`(pagesLoaded)` event gave me total number of pages loaded. Then `(pageChanged)` allowed me to find out if they scroll through all the pages.
